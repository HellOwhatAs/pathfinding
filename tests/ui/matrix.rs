use pathfinding::matrix;

fn main() {
    // Empty matrix must be rejected
    let _ = matrix!();
    // Single comma and multiple commas at the end must be rejected
    let _ = matrix!(,);
    let _ = matrix!( [1, 2], [3, 4], ,);
}
