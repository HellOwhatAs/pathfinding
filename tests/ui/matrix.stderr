error: no rules expected the token `,`
 --> tests/ui/matrix.rs:7:21
  |
7 |     let _ = matrix!(,);
  |                     ^ no rules expected this token in macro call

error: no rules expected the token `,`
 --> tests/ui/matrix.rs:8:38
  |
8 |     let _ = matrix!( [1, 2], [3, 4], ,);
  |                                      ^ no rules expected this token in macro call
